### PR TITLE
Auto delete bad things & add violent dimension to hive

### PR DESCRIPTION
**Pull request by dylanjha**
Two changes:

## 1. Auto-delete bad things

If an asset gets a `adult` score of `>= 0.95` then the following will happen:

1. Message posted to slack (see screenshot)
2. Row posted in Airtable in the "Auto Deleted" table
3. The asset's playback ID gets deleted (making the asset un-viewable and un-playable.

This felt like a good middle-ground. We're confident enough in the `adult >= 0.95` benchmark that we can confidently delete that content without anyone ever seeing it. If we get anecdotal reports of content being deleted unexpectedly then we can always go back and add a new playback ID and see if we were too agressive.

auto-deleted_2021-06-08_19-25-00

## 2. Add violent dimension to Hive

Below is the full list of classifications that Hive does. Some of these classifications are the top level ["head" models](https://docs.thehive.ai/reference#classification) -- like `general_nsfw` and `general_suggestive`. And then those head models have subclassifications (`yes_female_nudity` and `yes_male_nudity` are models that belong to the single model head).

As of now there is no `general_violence` head model, so we have to go off one of the other subclassifications. I went with `very_bloody`, but something else like `other_blood`, `a_little_bloody`, `knife_in_hand`, `gun_in_hand` might be good too (or we come up with some kind of strategy that does some kind combination or simply takes the max of these classifications that indicate violence.

<details>
<summary>Full list of available Hive dimensions</summary>

```
- animated
- text
- yes_overlay_text
- natural
- hybrid
- general_nsfw
- general_suggestive
- yes_female_underwear
- yes_male_underwear
- yes_sex_toy
- yes_female_nudity
- yes_male_nudity
- yes_female_swimwear
- yes_male_shirtless
- animated_gun
- gun_in_hand
- gun_not_in_hand
- culinary_knife_in_hand
- knife_in_hand
- knife_not_in_hand
- a_little_bloody
- other_blood
- very_bloody
- yes_pills
- yes_smoking
- illicit_injectables
- medical_injectables
- yes_nazi
- yes_kkk
- yes_middle_finger
- yes_terrorist
- yes_sexual_activity
- hanging
- noose
```

</details>

### CODE REVIEW

**Review by GeneticGenesis — APPROVED** [review]
This looks great to me! 

Let's add the config to Vercel and ship it!

File: lib/moderation-action.spec.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  expect(scopeAirtable.isDone()).toEqual(true);
+});
+
+test('returns false if no autoDelete happened', async () => {
+  const hiveScores = { adult: 0.89, suggestive: 0.11, violent: 0.234 };
+  const assetId = 'asset-123';
+  const playbackId = 'playback-123';
+  const isDeleted = await autoDelete({ assetId, playbackId, hiveScores });
+
+  expect(isDeleted).toEqual(false);
```
Comment (jsanford8) [inline lib/moderation-action.spec.ts:32]: Unit-test wise, might be good to validate that those API calls aren't made as well, but that's not necessary to hold the PR.

**Comment by GeneticGenesis** [comment]
(Just realised the config is already there and ready to go, it was just an update of the test config. Shipping this now!)

**Review by jsanford8 — APPROVED** [review]

File: lib/moderation-action.spec.ts (hunk 1 above)
Comment (dylanjha) [reply lib/moderation-action.spec.ts:32]: good shout!

**Pull request merged**